app/services/concerns/payloadable.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+module Payloadable
```

Comment:
Small nitpick: this name is weird, especially since it's used by services and not models…